### PR TITLE
HVS Intro

### PR DESCRIPTION
**Pull request by asheshvidyut**
HVS Introduction extends Consul setup

### CODE REVIEW

**Review by srahul3 — APPROVED** [review]

File: 7-hvs/README.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      "response_message": "Bello from ResponseService i-05506b6e36d25223a!"
+      }
+     ```
+10. **DIY**:
+   - Read the code and identify how to add `Tank yu` to the `minion_phrases`
+
+## Key Points
+- Dynamic service discovery: HelloService resolves ResponseService using Consul.
+- Centralized configuration via KV store.
+- Fault tollerant via consul circuit breaker
```
Comment (srahul3) [inline 7-hvs/README.md:167]: Can you review this, if it is applicable for HVS

File: 7-hvs/README.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      "minion_phrases": [
+         "Bello!",
+         "Poopaye!",
+         "Tulaliloo ti amo!"
+      ],
+      "response_message": "Bello from ResponseService i-05506b6e36d25223a!"
+      }
+     ```
+10. **DIY**:
+   - Read the code and identify how to add `Tank yu` to the `minion_phrases`
```
Comment (srahul3) [inline 7-hvs/README.md:162]: Can you review this, if it is applicable for HVS

File: 7-hvs/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+services:
+  hello-service:
+    image: ${TF_VAR_dockerhub_id}/helloservice:latest
+    container_name: helloservice
+    build: ./HelloService
+    environment:
+      RESPONSE_SERVICE_HOST: responseservice
+    ports:
+      - "9999:5000"
```
Comment (srahul3) [inline 7-hvs/docker-compose.yml:9]: Are we running this locally?, if not then we can use port 5000 awell.

File: 7-hvs/variables.tf [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  description = "Prefix used to name various infrastructure components. Alphanumeric characters only."
+  default     = "minion"
+}
+
+variable "response_service_count" {
+  description = "Number of response service instances to create"
+  default     = 1
+}
+
+variable "hcp_client_id" {}
```
Comment (srahul3) [inline 7-hvs/variables.tf:36]: add nullable = false

**Review by srahul3 — COMMENTED** [review]
Added few comments

File: 7-hvs/docker-compose.yml (hunk 3 above)
Comment (asheshvidyut) [reply 7-hvs/docker-compose.yml:9]: lets keep it same as consul

**Pull request merged**